src/umzug.ts

```diff
@@ -368,7 +368,7 @@ export class Umzug<Ctx> extends EventEmitter {
 			const paths = await globAsync(globString, { ...globOptions, absolute: true });
 			return paths.map(unresolvedPath => {
 				const filepath = path.resolve(unresolvedPath);
-				const name = path.basename(filepath, path.extname(filepath));
+				const name = path.basename(filepath);
```

Comment:
my only concern is that I have no idea why I did this is in the first place; it's more effort than doing nothing 🤷‍♂️

Reply:
Also by the way I slightly prefer `path.parse(filepath).name` instead of `path.basename(filepath, path.extname(filepath))` (it's the same thing)